Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -358,6 +360,10 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode
   /// Labels of coordinate system axes
   vtkSmartPointer<vtkStringArray> AxisLabels;
 
+  ///
+  /// Define if the view is mapped in the current layout with a Qwidget.
+  int MappedInLayout;
```

Comment:
Need to initialize, it can be done here or in the constructor.

Reply:
a nice, I forgot, thanks@